structure-maven-plugin/src/test/java/org/opennms/maven/plugins/StructureModuleTest.java

```diff
@@ -0,0 +1,37 @@
+package org.opennms.maven.plugins;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.apache.maven.plugin.testing.MojoRule;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class StructureModuleTest {
+    @Rule
+    public MojoRule rule = new MojoRule();
+
+    @Test
+    @Ignore
```

Comment:
Is there a reason that the test is ignored?

Reply:
Oops. Fixed.